openshift/patches/override-rekt-images.patch

```diff
@@ -33,7 +33,126 @@ index 80f37d08..860600a1 100644
    containers:
      - name: eventshub
 -      image: {{ .image }}
-+      image: registry.ci.openshift.org/openshift/knative-v1.2.3:knative-eventing-test-eventshub
++      image: registry.ci.openshift.org/openshift/knative-v1.2.1:knative-eventing-test-eventshub
```

Comment:
One more missing

Reply:
I do not think we build that here in this repo

Reply:
Ok, maybe we should.